docs/admin/deployment.rst

```diff
@@ -145,6 +145,10 @@ you create:
 ``MANUAL_SYNC``
    Optional. Enable Sync button in project Admin.
 
+``MEDIA_ROOT``
+   Optional. The absolute path of the "media" folder where the projects will be
+   cloned (it located next to the "pontoon" Python module by default).
```

Comment:
Nit: folder ~where~ the projects will be cloned **into** (it **is** located...

Reply:
Thank you for the english check :)